source/fuzz/replayer.cpp

```diff
@@ -80,6 +85,13 @@ Replayer::ReplayerResultStatus Replayer::Run(
       impl_->target_env, impl_->consumer, binary_in.data(), binary_in.size());
   assert(ir_context);
 
+  std::vector<uint32_t> last_good_binary;
+  std::vector<uint32_t> first_bad_binary;
```

Comment:
Is this used?

Reply:
No - good catch.  Initially I had been imagining saving out the last good and last bad binaries, but then I realised it would probably be more convenient to just have a place where a breakpoint could be set, so I didn't end up needing "first_bad_binary" (but forgot to remove it).